### ISSUE TITLE
Custom ports configuration

### ISSUE THREAD

**Issue by ll3N1GmAll** [h-mdm/remote-control#43]
I need to change the web portal and multiple other tcp/udp ports for headwind due to conflicting services behind the same public ip. 

Ports 80 & 443 need to be changed.   
Also, I need to change UDP ports 10000-10500 because they are already in use. 

I can leave ports 8089, 31000, & 8989 set to their defaults.

Where can I edit the build scripts and/or config files to accomplish this so everything gets built on custom ports? Thanks for your time and assistance! :)

**Comment by h-mdm** [comment]
The complete list of variables which could be changed (including ports!) can be found in deploy/config.defaults.yaml.

The variables are overridden in ./config.yaml.

**Comment by ll3N1GmAll** [comment]
> The complete list of variables which could be changed (including ports!) can be found in deploy/config.defaults.yaml.
> 
> The variables are overridden in ./config.yaml.

Thanks so much for providing that file! 

When you say "The variables are overridden in ./config.yaml", could you be a bit more explicit? I apologize; but I'm not clear on what you mean there. Thanks again! :)

**Comment by h-mdm** [comment]
I mean that you can specify the variables which you need to update, for example, web_https_port, in config.yaml, rather than changing the contents of the default configuration file.

**Comment by ll3N1GmAll** [comment]
> I mean that you can specify the variables which you need to update, for example, web_https_port, in config.yaml, rather than changing the contents of the default configuration file.

Excellent, thanks so much!

**Comment by ll3N1GmAll** [comment]
It seems now that the certbot task fails now that nginx is not running on port 80. However, another apache server is running on that VM on port 80. Is there a way to change the port that certbot is using by default? Is there another certificate option documented instead of certbot?

**Comment by ll3N1GmAll** [comment]
Also, after adding the variables from deploy/config.defaults.yaml to config.yaml with my custom settings and then running the install.sh, I see no services running on the VM with the ports I set. In fact, nginx doesn't appear to be running at all. Is this all in a docker instance and this is why I am not able to see it? I cannot reach the server from any clients on the custom ports I specified in config.yaml.
